webpack/assets/javascripts/react_app/components/AutoCompleteSearch/AutoCompleteSelector.js

```diff
@@ -0,0 +1,7 @@
+import { createSelector } from 'reselect';
+import { renderMenu } from './helpers';
+
+export const menuSelector = createSelector(
+  ({ autocomplete: { results, searchQuery } }) => ({ results, searchQuery }),
+  menuProps => renderMenu(menuProps),
```

Comment:
Does it make sense to have a selector that return component?
Shouldn't it return a json that a component receive?